apiserver/authentication/jwt/doc.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// Package jwt provides an authentication mechanism whereby a JAAS-registered
+// Juju controller can accept a JAAS JWT (JSON Web Token) as the means to
+// declare a user and their permissions.
+//
+// This mechanism kicks in if, after setting up the JWT in JAAS, you bootstrap your
+// controller with the 'login-token-refresh-url' controller config set to
+// the JAAS JWT refresh endpoint.
+//
+// Juju uses a [github.com/juju/juju/apiserver/authentication.PermissionDelegator]
+// instance to look up what permissions a user has. For non-JWT scenarios, these
+// come from the Juju permissions model. However, when a JWT is used, the Juju
+// permission model is empty, and the permissions are retrieved from the JWT.
```

Comment:
Insert new line.

Reply:
Really? That last sentence does belong to the paragraph IMO. It's all talking about how the JWT is created and used. Will change if you see there's a clearly separate purpose to having it separate.

Reply:
I was still until the influence of my previous rewrite, where this paragraph was focused on what goes on beneath the surface and the next focused on the impact. But now that the impact is split between the first paragraph and the next sentence here, I'm not sure anymore. So let's think about that first (it's also the question I had in my comment on the next sentence).